### ISSUE TITLE
Card sorting should default to ascending

### ISSUE THREAD

**Issue by declension** [cgorski/general-games#1]
Hi. First - thanks for a great library, it's just what I was looking for :+1: 

Whilst exploring I was surprised to notice that card sorting seems to be descending by default.

### Example 
From a `stack repl` in the project directory:
```haskell
λ> import Data.List
λ> ranks = [Two, Ace, Three]
λ> sort ranks                            -- Use Prelude-based sort on the ranks alone
[Ace, Two, Three]
λ> cards = flip PlayingCard Clubs <$> ranks
λ> sortCardsBy AceLowRankOrder $ cards   -- Default library sorting ordering is descending
[Three of Clubs,Two of Clubs,Ace of Clubs]
λ> sortCardsBy AceHighRankOrder $ cards  -- Also uses descending
[Ace of Clubs,Three of Clubs,Two of Clubs]
```

IMO it's more intuitive to maintain ascending as the default.

**Comment by cgorski** [comment]
I agree with you. I'll look into it for the next release. Thanks for finding this!

**Comment by cgorski** [comment]
Fixed in release 1.1.1
